src/plugins/autocompletion_providers/Process.ts

```diff
@@ -0,0 +1,188 @@
+import {PluginManager} from "../../PluginManager";
```

Comment:
Our convention is to use the name of the command as the name of the file. Please changes this file to `Ps.ts`